### PR TITLE
update Sentry Basics for alert wizard changes

### PR DESCRIPTION
**Pull request by imatwawana**
Updates the Alerts and Notifications content in Sentry Basics for alert wizard changes

### CODE REVIEW

File: src/docs/product/sentry-basics/guides/alert-notifications/index.mdx [hunk 1]
```diff
@@ -10,13 +10,13 @@ Both Sentry's alerts and notifications provide real-time visibility into errors
...
+When you create a new project in [sentry.io](https://sentry.io), you can choose from a few default alerts, or choose to create alerts at a later time. Choosing a default alert, such as getting notified only once when an error is first seen, is useful for catching new errors in real time, but means Sentry will not alert you about subsequent instances of the same error. With only the default alert rule in place, you might be losing visibility into critical issues building up in your code.
 
-![Choice between Metric Alert or Issue Alert](create-new-alert-rule.png)
+Instead, we recommend you create your own alerts to provide you with the information that your team needs at a useful frequency. To create your own alert, navigate to **Alerts** and click "Create New Alert". Select your project, and then select from the alerting options under "Errors" or "Performance".
```
Comment (PeloWriter) [inline src/docs/product/sentry-basics/guides/alert-notifications/index.mdx:15]:
```suggestion
Instead, we recommend creating your own alerts to provide information that your team needs at a frequency that is useful to your team's triage process. To create your own alert, navigate to **Alerts** and click "Create New Alert". Select your project, then select from the alerting options under "Errors" or "Performance".
```

File: src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
 
-## 1. Choose Errors or Transactions
+## 1. Choose Errors or Performance
 
-In the Alert Conditions section, choose between [Errors](/platforms/javascript/usage/#capturing-errors) or [Transactions](/product/performance/). Errors provide data around a break, such as new issues or total errors in your project. Transactions provide data on the performance of your application, such as [latency](/product/performance/metrics/#latency) and [failure rate](/product/performance/metrics/#failure-rate).
+Select what you want to be alerted about from a number of preset alerts under "Errors" or "Performance". Your selection determines some of the alert properties, including whether you're alerting on [errors](/platforms/javascript/usage/#capturing-errors) or [transactions](/product/performance/). Errors provide data around a break, such as new issues or total errors in your project. Transactions provide data on the performance of your application, such as [latency](/product/performance/metrics/#latency) and [failure rate](/product/performance/metrics/#failure-rate). You can also choose to create a custom metric.
```
Comment (PeloWriter) [inline src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx:13]: general q: we use the phrase "alerting on", which focuses on what Sentry does. I think we might want to switch to "alerted to", which focuses on the result of the users's set up. Your call.

**Review by PeloWriter — APPROVED** [review]

File: src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx (hunk 2 above)
Comment (imatwawana) [inline src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx:13]:
```suggestion
Select what you want to be alerted about from a number of preset alerts under "Errors" or "Performance". Your selection determines some of the alert properties, including whether you're alerted about [errors](/platforms/javascript/usage/#capturing-errors) or [transactions](/product/performance/). Errors provide data around a break, such as new issues or total errors in your project. Transactions provide data on the performance of your application, such as [latency](/product/performance/metrics/#latency) and [failure rate](/product/performance/metrics/#failure-rate). You can also choose to create a custom metric.
```

**Review by imatwawana — COMMENTED** [review]
Wording edits

**Comment by imatwawana** [comment]
Part of work on issue 3379
https://github.com/getsentry/sentry-docs/issues/3379

File: src/docs/product/sentry-basics/guides/alert-notifications/index.mdx [hunk 3]
```diff
@@ -10,13 +10,13 @@ Both Sentry's alerts and notifications provide real-time visibility into errors
...
 
-![Choice between Metric Alert or Issue Alert](create-new-alert-rule.png)
+Instead, we recommend creating your own alerts to provide information that your team needs at a frequency that is useful to your team's triage process. To create your own alert, navigate to **Alerts** and click "Create New Alert". Select your project, then select from the alerting options under "Errors" or "Performance".
 
-Metric alerts are usually more straightforward than issue alerts because you know upfront what you're monitoring. In contrast, issue alerts power a triage workflow — you cast a wide net for potential problems and review them as they happen.
+![Choice between alerts about Errors, Performance, or a Custom Metric](create-alert-rule)
```
Comment (imatwawana) [inline src/docs/product/sentry-basics/guides/alert-notifications/index.mdx:17]:
```suggestion
![Choice between alerts about Errors, Performance, or a Custom Metric](create-alert-rule.jpg)
```

File: src/docs/product/sentry-basics/guides/alert-notifications/issue-alerts.mdx [hunk 4]
```diff
@@ -4,20 +4,23 @@ sidebar_order: 2
...
 
 Each environment typically necessitates different levels of urgency and workflow. The urgency and workflows you apply to Production alerts might differ from those you apply to alerts originating from your QA environment, for example.
 
-## 2: Give the Alert Rule a Name
+## 2: Name the Alert & Add a Team
 
 Manage your alert rule by giving it a descriptive name. A descriptive alert rule name specifies the team affected and the topic of the alert. For example, "Frontend Latency," "Backend Failure Rate," or "Billing Apdex."
 
+You can also set the team that should be notified of this alert. Only teams that you're a member of are available for you to select. The team that you set here owns the alert and is automatically assigned to it.
```
Comment (imatwawana) [inline src/docs/product/sentry-basics/guides/alert-notifications/issue-alerts.mdx:22]: @iProgramStuff - here's the change we talked about; does this capture it?

**Comment by adhiraj** [comment]
lgtm, feel free to merge whenever

**Comment by imatwawana** [comment]
Completes part of [issue 3379](https://github.com/getsentry/sentry-docs/issues/3379)

**Pull request merged**